NUnitTestProject1/Tests/TestClass.cs

```diff
@@ -108,19 +113,19 @@ public void SortDropdownTest()
         }
 
         [Test]
+        [Parallelizable(ParallelScope.Self)]
```

Comment:
Try to use FluentAssetrions instead of Nunit assets

https://fluentassertions.com/